src/phoenix/server/api/subscriptions.py

```diff
@@ -480,9 +478,11 @@ def _llm_token_counts(usage: "CompletionUsage") -> Iterator[Tuple[str, Any]]:
 
 
 def _input_value_and_mime_type(input: ChatCompletionInput) -> Iterator[Tuple[str, Any]]:
-    assert any(field.name == (api_key := "api_key") for field in fields(ChatCompletionInput))
+    assert (api_key := "api_key") in (input_data := jsonify(input))
+    input_data = {k: v for k, v in input_data.items() if k != api_key}
+    assert api_key not in input_data
```

Comment:
```suggestion
    input_data = {k: v for k, v in jsonify(input).items() if not k.endswith("api_key")}
```

Reply:
Just because of the sensitive nature of writing API keys to disk, I want to make sure we get an unmissable error if the API changes in the future (e.g., if `api_key` is renamed or refactored to a different location). I tried figuring out a way to get type error if that kind of change happens, but did not think of one. Any other thoughts?